fwdpy11/src/functions/add_mutation.cc

```diff
@@ -158,6 +158,46 @@ namespace
     }
 }
 
+std::int64_t
```

Comment:
This is close, but not right yet.  We are ignoring the case when the node's time cannot be represented exactly as an int64 and the node's parent's time is such that no int64 exists between the two node times.